modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/saml/AbstractSAMLSSOTestCase.java

```diff
@@ -407,14 +420,6 @@ public void deleteApplication(String appName) throws Exception {
         lastNameClaimMapping.setRemoteClaim(lastNameClaim);
         claimMappingList.add(lastNameClaimMapping);
 
-        Claim emailClaim = new Claim();
```

Comment:
What would be the reason behind removing email claim?

Reply:
This was added again with https://github.com/wso2/product-is/pull/11407/commits/4fe18ee0052418875fffa6a22be116143ea73ebb. Claim configs were separately added and updated in the SAMLSSOConsentTestCase.